spec/system/admin/enterprises/index_spec.rb

```diff
@@ -35,11 +35,11 @@
     end
 
     context "editing enterprises in bulk" do
-      let!(:d){ create(:distributor_enterprise, sells: 'none') }
-      let!(:d_manager) { create(:user, enterprise_limit: 1) }
+      let!(:distributor){ create(:distributor_enterprise, sells: 'none') }
+      let!(:mani) { create(:user, enterprise_limit: 1) }
```

Comment:
I don't understand the `mani` reference..

Reply:
Hm, it's just a name. At some point I found it helpful to use real names for users as suggested by Myriam:

* https://community.openfoodnetwork.org/t/personas-lets-name-our-users/1135 <- that's from the first gathering a bit over five years ago Kirsten was referring to on Monday.

But that probably works better with more common names. Do you have any suggestions?

Reply:
Oh wow, they're nice pictures. In this case I guess a distributor would be a Mary or Shannon? A common name would help, but only for those whom it is common to..

Hmm I appreciate a bit of creativity and fun in specs, but I would try to avoid indirection most of the time. I would find `distributor_user` or simply `user` much easier to follow. 

🤷  Naming things is hard.

Reply:
Agree with @dacook.

I usually add real names (more or less common) where these can actually be seen and found by Capybara in the app (for example a first- and last-name on the customers page or so).

Since these are variables I'd say `distributor_user` or  `user` are easier to follow and perhaps clearer on what type of entity we're aiming at, when we create it in the spec.

Reply:
Okay, I think that I will rename it to `manager` because that's more specific than `user` in this context. It's where the spec begins. But what I find difficult with this is that roles can change over the course of the example. This manager is becoming an owner in one example. It's okay here because the owner can still manage as well. If someone adds a test for the removal of the user then we may have a problem with the naming. But we can worry about that then.